src/vs/workbench/contrib/terminal/browser/links/terminalLinkOpeners.ts

```diff
@@ -36,81 +37,16 @@ export class TerminalLocalFileLinkOpener implements ITerminalLinkOpener {
 		if (!link.uri) {
 			throw new Error('Tried to open file link without a resolved URI');
 		}
-		const lineColumnInfo: ILineColumnInfo = this.extractLineColumnInfo(link.text, link.uri);
+		const linkSuffix = getLinkSuffix(link.text);
```

Comment:
what happens if these are undefined?

Reply:
oh I see they default to 1

Reply:
Now that you say that, it's probably best to pass in undefined in case the file is already open and at a position we shouldn't replace the cursor position.